nengo/cache.py

```diff
@@ -284,16 +285,21 @@ def _check_legacy_file(self):
         legacy_file = os.path.join(self.cache_dir, self._LEGACY)
         if os.path.exists(legacy_file):
             with open(legacy_file, 'r') as lf:
-                version = int(lf.read().strip())
+                text = lf.read()
+            try:
+                lv, pp = tuple(int(x.strip()) for x in text.split('.'))
```

Comment:
Is there any chance we would have periods in the legacy version (e.g. `1.1`)?  Could just write the `lv`/`pp` to separate lines if we wanted to avoid any issues.